src/domain/alerts/alerts.repository.ts

```diff
@@ -51,33 +52,69 @@ export class AlertsRepository implements IAlertsRepository {
       return this._notifyUnknownTransaction(chainId, log);
     }
 
-    const safe = await this.safeRepository.getSafe({
-      chainId,
-      address: decodedEvent.args.to,
+    try {
+      const safeAddress = decodedEvent.args.to;
+
+      const newSafe = await this.predictNewSafeSetup({
+        chainId,
+        safeAddress,
+        decodedTransactions,
+      });
+
+      return this._notifyNewSafeSetup({ chainId, safeAddress, newSafe });
+    } catch {
+      return this._notifyUnknownTransaction(chainId, log);
+    }
+  }
+
+  private decodeTransactionAdded(data: Hex) {
```

Comment:
Nitpick: could we prefix private functions with `_`? I know it's not a standard through all the codebase, but at least to stay consistent within this file.

Reply:
Adjusted in ab0c720fca0ecfeba81054f22fc9ad347c96765d.